src/Boot/DrupalBoot.php

```diff
@@ -21,7 +21,14 @@ public function findUri($root, $cwd): string
                 return basename($siteDir);
             }
         }
-        return 'default';
+        // attempt to detect a domain name from the path, matches www.example.org or example.net
+        $detected_uri = preg_match('/[a-z]*\.?[a-z]+\.[a-z]+/', dirname(__FILE__), $matches);
```

Comment:
I would think we could use $root, and not `dirname(__FILE__)`

Reply:
Great suggestion, the PR is updated to use $root. Thanks!